.travis.yml

```diff
@@ -8,10 +12,25 @@ php:
   - 7.0
   - hhvm
 
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
 matrix:
-  allow_failures:
-    - php: 7.0
+  fast_finish: true
+  include:
+    - php: 5.3
+      env: COMPOSER_FLAGS="--prefer-lowest"
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*@dev SYMFONY_DEPRECATIONS_HELPER=strict
+    - php: 5.6
+      env: SYMFONY_VERSION=3.0.*@dev SYMFONY_DEPRECATIONS_HELPER=strict DEPENDENCIES=dev COMPOSER_FLAGS="--prefer-stable"
+
+before_install:
+    - composer self-update
+    - 'if [ "$SYMFONY_VERSION" != "" ]; then sed -i "s/\"symfony\/\([^\"]*\)\": \"[^\"]*\"/\"symfony\/\1\": \"$SYMFONY_VERSION\"/g" composer.json; fi'
```

Comment:
If I use ``composer require``, I have to copy the symfony dependencies in the travis file...
This command allows to use new symfony dependencies without updating the travis file.